tests/func-tests/virtual_machines_test.go

```diff
@@ -21,27 +24,52 @@ var _ = Describe("Virtual Machines", func() {
 	client, err := kubecli.GetKubevirtClient()
 	testscore.PanicOnError(err)
 
+	var (
+		workloadsNode      *corev1.Node
+		checkNodePlacement = false
+	)
+	workloadsNodes, err := client.CoreV1().Nodes().List(k8smetav1.ListOptions{
+		LabelSelector: "node.kubernetes.io/hco-test-node-type==workloads",
+	})
+
+	if err == nil && workloadsNodes != nil && len(workloadsNodes.Items) > 0 {
+		checkNodePlacement = true
+		workloadsNode = &workloadsNodes.Items[0]
```

Comment:
In case there is more than one workloadNode, is there a reason you're assuming the VMI will be scheduled on the first node in the `workloadsNodes` list? (refer to line 63)

Reply:
Right. But this is a unique label that is only set in known tests. We know there is only one (or zero)